public/javascripts/embed.js

```diff
@@ -3,80 +3,78 @@
 // This happens on the child window
 var Pender = {};
 
-(function($) {
```

Comment:
Question: If I understand correctly, the only usage of jQuery is this `$` passed as a parameter, which is not even used, right, @vasconsaurus ? If so, I suggest we just delete the `$` and keep the rest as is (keep the `function() {`).

Reply:
The form `(function(){...})()` is a function literal that is executed immediately. What this means is that you have defined a function and you are calling it immediately.

This form is useful for information hiding and encapsulation since anything you define inside that function remains local to that function and inaccessible from the outside world. So, mostly, encapsulation.